master/custom/factories.py

```diff
@@ -259,6 +259,12 @@ class UnixBuildWithoutDocStrings(UnixBuild):
     configureFlags = ["--with-pydebug", "--without-doc-strings"]
 
 
+class UnixBigmemBuild(UnixBuild):
+    buildersuffix = ".bigmem"
+    testFlags = ["-M60g", "-j4", "-uall,extralargefile"]
+    factory_tags = ["aarch64", "bigmem"]
```

Comment:
The `aarch64` tag doesn't really belong here; there's nothing specific to `aarch64` in this builder factory (it would work perfectly well with a similarly beefy x86_64 machine).  We do need to fix how tags are assigned to include tags from the worker as well, though.

Reply:
ok, I'll remove it then.